pandas/_typing.py

```diff
@@ -28,3 +35,4 @@
 Scalar = Union[str, int, float]
 Axis = Union[str, int]
```

Comment:
can Axis be more specifically [1, 2, "columns", "index"]?

Reply:
yes. updated.

need a few casts though!

Reply:
@jbrockmendel the Literal type has been removed for now. so marking this as resolved as no longer applicable.